delta-inputs/s3-to-stash.ts

```diff
@@ -41,8 +65,44 @@ s3Stream.on('finish', () => process.exit(0))
 const deltaWriteStream = stashDB.deltaWriteStream()
 deltaWriteStream.on('deltaProcessed', () => tsvRowCount++)
 
+class SetSelfVesselId extends Transform {
+  readonly selfId: string
+  readonly isSelf: (delta: any) => boolean
+  readonly importAll: boolean
+
+  constructor(selfVesselId: string, importAll: boolean | undefined) {
```

Comment:
Optional parameters in TS can be denoted using `?` . For example `importAll?: boolean` leads to same type (`boolean | undefined`) here.